src/cli.rs

```diff
@@ -204,6 +204,22 @@ pub enum Commands {
         #[arg(long, value_enum, default_value_t = DEFAULT_LEVEL)]
         level: AaLevel,
     },
+
+    /// Delete genome(s) from a database (input: one id per line)
+    Delete {
+        /// Sketching database basename (so without .skm or .skd)
+        #[arg(required = true)]
+        db: String,
+
+        /// Input file with IDs to delete (one ID per line)
+        #[arg(required = true)]
+        genome_ids: String,
```

Comment:
I think `sample_ids` or just `samples` would be better (as they may be genes, not genomes)